src/components/ActionsBlock.ts

```diff
@@ -5,8 +5,10 @@ import { ContainerProps } from './ContainerProps';
 export type ActionsBlockProps = BlockProps & ContainerProps<any>;
 
 export const ActionsBlock: Block<ActionsBlockProps, ActionsBlockSpec> = ({
+  blockId,
   children,
 }) => ({
   type: 'actions',
+  block_id: blockId,
```

Comment:
hmmm what about other blocks? they already set this? or should you apply same fix?

Reply:
all the other blocks i found already had this